src/tyr/route_serializer_osrm.cc

```diff
@@ -283,23 +195,36 @@ json::MapPtr geojson_shape(const std::vector<PointLL> shape) {
   return geojson;
 }
 
+// Return a JSON array of OpenLR 1.5 line location references for each edge of a map matching
+// result. For the time being, result is only non-empty for auto costing requests.
+json::ArrayPtr route_references(const valhalla::TripRoute& route, const valhalla::Options& options) {
```

Comment:
can we add this the valhalla serializer as well

Reply:
Ah, I knew you were going to say that.